### DISCUSSION TITLE
升级到zotero7以后style插件无法使用了

### DISCUSSION THREAD

**Discussion by aaalu26** [MuiseDestiny/zotero-style#1078] [Q&A]
从官网下载安装的windows 64-bit版本的zotero7，直接覆盖原来的6，安装以后style插件无法使用。重新下载新版本的插件安装后依旧无法使用。如图：
![29706](https://github.com/user-attachments/assets/f0836633-ad55-4568-ba4f-b99e9f2c760f)

**Comment by Fengyian** [comment]
我也是，不知道咋整

**Comment by lbc0801** [comment]
+1
